src/controllers/accounts/helpers.js

```diff
@@ -91,7 +91,6 @@ helpers.getUserDataByUserSlug = async function (userslug, callerUID) {
 	});
 
 	userData.sso = results.sso.associations;
-	userData.banned = userData.banned === 1;
```

Comment:
The banned property is used by themes to display a banned message for example https://github.com/NodeBB/nodebb-theme-persona/blob/master/templates/account/profile.tpl#L8-L16

Reply:
It is ensured to be present at that point, and coming as a boolean. Better to be explicit :thumbsup:. I think this would suffice:

`Boolean(userData.banned)`